app/src/main/java/mobi/visamate/view/DisclaimerFragment.kt

```diff
@@ -16,6 +18,15 @@ class DisclaimerFragment : Fragment() {
                               savedInstanceState: Bundle?): View? {
         val rootView = inflater.inflate(R.layout.fragment_disclaimer, container, false)
```

Comment:
Another polemic comment: could we agree not to use Fragments? I removed all fragments from previous apps and it was great.

Reply:
Thats might be fine for mobile but for tablet it doesn't.
And the second objective of the app is for everyone to practice and learn too, so it will be more challenging and fun ;)

Reply:
Yeah, for tablet support I used custom views as view components. Fragments are overkill most of the time, but I'm ok to use too. ;)